src/protocols/rdp/channels/rail.h

```diff
@@ -20,7 +20,34 @@
 #ifndef GUAC_RDP_CHANNELS_RAIL_H
 #define GUAC_RDP_CHANNELS_RAIL_H
 
+#include "config.h"
+
 #include <freerdp/freerdp.h>
+#include <freerdp/window.h>
+
+#ifdef FREERDP_RAIL_CALLBACKS_REQUIRE_CONST
+/**
+ * FreeRDP 2.0.0-rc4 and newer requires the final argument for all RAIL
```

Comment:
Isn't it the last 2 arguments? That's what I see in the actual function signature.

Reply:
Well, it depends on how many arguments the functions have. To your point, the `WindowUpdate` callback has two arguments, both of which need to be `const`; however, the `ServerHandshake` and `ServerHandshakeEx` callbacks only have a single argument that is `const`.

I've tweaked the comment via rebase.